Source/Scene/processPbrMaterials.js

```diff
@@ -602,7 +602,7 @@ import ModelUtility from './ModelUtility.js';
                 fragmentShader += '    vec3 n = ng;\n';
             }
             if (material.doubleSided) {
-                fragmentShader += '    if (!gl_FrontFacing)\n';
+                fragmentShader += '    if (gl_FrontFacing == false)\n';
```

Comment:
Maybe add a comment here explaining it works around an issue and link to this PR?  Otherwise a developer might "clean up" the code here and accidentally go back to the simpler ! check.

Reply:
Ah good suggestion, updated.